store/models.py

```diff
@@ -201,6 +202,13 @@ class Meta:
         blank=True
     )
 
+    metadata = models.TextField(
+        verbose_name=_("Metada"),
+        max_length=1000,
```

Comment:
As you can see on this documentation, `max_length` is not enforced on database level. It's just for front-end generation purpose when we don't use DRF on top of it.

https://docs.djangoproject.com/en/2.2/ref/models/fields/#textfield